torch/utils/tensorboard/summary.py

```diff
@@ -383,7 +383,7 @@ def make_histogram(values, bins, max_bins=None):
         limits = new_limits
 
     # Find the first and the last bin defining the support of the histogram:
-    cum_counts = np.cumsum(np.greater(counts, 0, dtype=np.int32))
+    cum_counts = np.cumsum(np.greater(counts, 0).astype(np.int32))
```

Comment:
`np.greater(counts, 0, dtype=np.int32)` just returns an array of dtype np.bool, right? If so we can just delete the dtype=np.int32 argument